client/app/account/user/credentials/read/user-credentials-read.html

```diff
@@ -0,0 +1,46 @@
+<div data-ng-controller="UserAccount.controllers.credentials.read as $ctrl">
+
+    <div data-wizard
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="resetAction"
+         data-wizard-title="i18n.user_credentials_read_modal_title">
+
+        <div data-wizard-step>
+
+            <p data-ng-bind="$ctrl.credential.name"></p>
+            <table class="table table-hover">
+                <thead>
+                    <tr>
+                        <th scope="col"
+                            data-i18n-static="user_credentials_permissions_method">
+                        </th>
+                        <th scope="col"
+                            data-i18n-static="user_credentials_permissions_path">
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr data-ng-repeat="rule in $ctrl.credential.rules track by $index">
+                        <td>
+                            <span class="label"
+                                  data-ng-class="{
+                                      'label-danger': rule.method === 'DELETE',
```

Comment:
Why is `DELETE` considered a "danger" ? I understand it's only a matter of color but it's a bit weird, was this CX validated ?